bottomless/src/replicator.rs

```diff
@@ -239,12 +240,13 @@ impl Replicator {
         let db_path = db_path.into();
         let db_name = {
             let db_id = options.db_id.unwrap_or_default();
-            let name = match db_path.rfind('/') {
-                Some(index) => &db_path[index + 1..],
+            let name = match db_path.find('/') {
+                Some(index) => &db_path[..index],
```

Comment:
This change is because previously we used the last database path segment, which was always `/data`. I've changed that to use first segment, but it may have it's own issues depending on what will we provide as a path.

Any thoughts @psarna ?